### ISSUE TITLE
[BUG/ISSUE] Issue in calculating DryDep diagnostics for GCHP

### ISSUE THREAD

**Issue by msulprizio** [geoschem/gcpy#121]
There seems to be an issue in the DryDep diagnostic for GCHP. This was discovered when generating the Ox budget table for the GCHP 13.0.0 1-year benchmark. From that routine in GCPy, we get a dry deposition sink of 849 Tg Ox/yr for GCClassic and 9742 Tg Ox/year for GCHP.

Using gfortran 9.3, I ran 1-hour integration tests for GCClassic (commit: [`c8f8e057`](https://github.com/geoschem/GCClassic/commit/c8f8e0576a432e7f964a22c87d04e13815235089), simulation: 4x5 MERRA-2 benchmark) and GCHP (commit: 1e3150d848f11ca3d45802a84ac6dfa41c65f191, simulation: c48 MERRA-2 benchmark).  Using compare_diags.py from GCPy, we get:

```
Variable               Ref=GCC_13.1.0-alpha.7   Dev=GCHP_13.1.0-alpha.7  Dev-Ref
DryDepChm_O3         : 0.0                    | 0.0                    | 0.0 
DryDepMix_O3         : 204570940000000.0      | 958071800000000.0      | 753500875128832.0 
DryDep_O3            : 204570940000000.0      | 958071800000000.0      | 753500875128832.0 
DryDepChm_HNO3       : 0.0                    | 0.0                    | 0.0 
DryDepMix_HNO3       : 11319096000000.0       | 65056662000000.0       | 53737565454336.0 
DryDep_HNO3          : 11319096000000.0       | 65056662000000.0       | 53737565454336.0
```

[single_level_comparison.pdf](https://github.com/geoschem/GCHP/files/6467434/single_level_comparison.pdf)

**Comment by lizziel** [comment]
Was this issue fixed for the 13.2 benchmark?

**Comment by msulprizio** [comment]
No, this is still not resolved. When generating the Ox budget tables for the 1-year benchmarks for 13.2.0 we get:

```
GEOS-Chem Classic 13.2.0:

  Ox SINKS                Tg Ox a-1
  -----------------      ----------

  * Dry deposition         844.8246

  * Wet deposition
      Convective            16.1201
      Large-Scale           59.8803
                         ----------
      Total Wetdep        76.000428
```

```
GCHP 13.2.0:

  Ox SINKS                Tg Ox a-1
  -----------------      ----------

  * Dry deposition        9814.7573

  * Wet deposition
      Convective            17.2765
      Large-Scale           57.6045
                         ----------
      Total Wetdep        74.881005
```

**Comment by lizziel** [comment]
Okay. @laestrada, would you be able to take this one? It's a lingering GCPy bug affecting 1-year benchmarks so we should take care of it sooner rather than later.

**Comment by lizziel** [comment]
We excluded the budget diagnostic tables in the 1-month benchmark but we should still calculate global Ox budget for the 1-year benchmarks, such as [this one](http://ftp.as.harvard.edu/gcgrid/geos-chem/1yr_benchmarks/13.3.0-rc.0/GCClassic/FullChem/BenchmarkResults/Tables/GCC_13.3.0.O3_budget_troposphere_2019.txt). I think the 1-year HISTORY.rc outputs just the subset of budget diagnostics needed to calculate it so we avoid the memory and performance issues of turning them on for all species.

**Comment by lizziel** [comment]
Is this still an issue that needs to be fixed?

**Comment by msulprizio** [comment]
I believe it is. I think I have the Ox budget tables disabled for GCHP 1-year benchmarks because of this, but I will confirm when I regenerate the GCHP 1-year benchmark output for 14.0.0-rc.3.
